docs/pages/database-access/guides/sql-server-ad.mdx

```diff
@@ -0,0 +1,315 @@
+
+<Tabs>
+  <TabItem label="Ubuntu">
+    ```bash
+    $ sudo apt-get update
+    $ sudo apt-get -y install sssd realmd krb5-user samba-common packagekit adcli
+    ```
+  </TabItem>
+  <TabItem label="RHEL / CentOS 7">
+    ```bash
+    $ sudo yum -y update
+    $ sudo yum -y install sssd realmd krb5-workstation samba-common-tools
+    ```
+  </TabItem>
+</Tabs>
+
+Edit `/etc/krb5.conf` to disable reverse DNS resolution and set the default
+realm:
+
+```ini
+[libdefaults]
+default_realm = EXAMPLE.COM
+rdns = false
+```
+
+Join the realm:
+
+```bash
+$ sudo realm join -v -U admin@EXAMPLE.COM example.com
+...
+ * Successfully enrolled machine in realm
+```
+
+<Admonition type="warning">
+  Note that the realm name in `admin@EXAMPLE.COM` **must** be capital case,
+  otherwise the node might not be able to join.
+</Admonition>
+
+To confirm the node has joined the realm, use the `realm list` command:
+
+```bash
+$ sudo realm list
+example.com
+  type: kerberos
+  realm-name: EXAMPLE.COM
+  domain-name: example.com
+  configured: kerberos-member
+  server-software: active-directory
+  client-software: sssd
+  ...
+```
+
+## Step 4/7. Create keytab file
+
+Teleport requires a keytab file to obtain Kerberos service tickets from your
+Active Directory for authentication with SQL Server. The easiest way to generate
+it is to use the `adutil` Linux CLI utility.
+
+Install `adutil` on the Linux node you have joined to your Active Directory
+domain:
+
+<Tabs>
+  <TabItem label="Ubuntu 18.04">
+    ```bash
+    $ curl https://packages.microsoft.com/keys/microsoft.asc | sudo apt-key add -
+    $ sudo curl https://packages.microsoft.com/config/ubuntu/18.04/prod.list | sudo tee /etc/apt/sources.list.d/msprod.list
+    $ sudo apt-get update
+    $ sudo ACCEPT_EULA=Y apt-get install -y adutil
+    ```
+  </TabItem>
+  <TabItem label="Ubuntu 20.04">
+    ```bash
+    $ sudo wget -qO /etc/apt/trusted.gpg.d/microsoft.asc https://packages.microsoft.com/keys/microsoft.asc
+    $ sudo curl https://packages.microsoft.com/config/ubuntu/20.04/prod.list | sudo tee /etc/apt/sources.list.d/msprod.list
+    $ sudo apt-get update
+    $ sudo ACCEPT_EULA=Y apt-get install -y adutil
+    ```
+  </TabItem>
+  <TabItem label="RHEL / CentOS 7">
+    ```bash
+    $ sudo curl -o /etc/yum.repos.d/msprod.repo https://packages.microsoft.com/config/rhel/8/prod.repo
+    $ sudo ACCEPT_EULA=Y yum install -y adutil
+    ```
+  </TabItem>
+</Tabs>
+
+Log into your Active Directory using the `kinit` command:
+
+```bash
+$ kinit admin@EXAMPLE.COM
+```
+
+Use the `adutil keytab create` command to generate keytab entries for each
+Active Directory user that will be connecting to the SQL Server database:
+
+```bash
+$ adutil keytab create teleport.keytab alice
+$ adutil keytab create teleport.keytab bob
+```
+
+You will be prompted to enter each user's password. All keytab entries will
+be merged into the same `teleport.keytab` file.
+
+<Admonition type="note" title="SPN">
```

Comment:
I think we should use a more descriptive title so users don't skip over this box.